orchagent/portsorch.cpp

```diff
@@ -112,6 +113,26 @@ static map<string, int> autoneg_mode_map =
     { "off", 0 }
 };
 
+static map<string, int> link_training_mode_map =
+{
+    { "on", 1 },
```

Comment:
Can't we use a bool instead of int? This will make code much simpler

Reply:
In order to distinguish 'NOT CONFIURED' and 'OFF', we need -1 for this, and this is the same approach for the AutoNeg in the current implementation of SONiC